xml/depl_nodes.xml

```diff
@@ -1457,7 +1457,7 @@ Verify return code: 18 (self signed certificate)
      You may make this available to &o_dash; by setting <guimenu>multi_domain_support</guimenu> to <guimenu>true</guimenu> in the &o_dash; &barcl;.
  </para>
  <para>
-Users in the LDAP-backed domain have to know the name of the domain in order to authenticate, and must use the  &o_ident; v3 API endpoint. (See the &ostack; manuals, <link xlink:href="https://docs.openstack.org/keystone/latest/admin/identity-domain-specific-config.html">Domain-specific Configuration</link> and <link xlink:href="https://docs.openstack.org/keystone/latest/admin/identity-integrate-with-ldap.html">Integrate Identity with LDAP</link>, for additional details.)
+Users in the LDAP-backed domain have to know the name of the domain in order to authenticate, and must use the  &o_ident; v3 API endpoint. (See the &ostack; manuals, <link xlink:href="https://docs.openstack.org/keystone/rocky/admin/identity-domain-specific-config.html">Domain-specific Configuration</link> and <link xlink:href="https://docs.openstack.org/keystone/rocky/admin/identity-integrate-with-ldap.html">Integrate Identity with LDAP</link>, for additional details.)
```

Comment:
We should reaaallyyy be wrapping this text but it's hardly important right now

Reply:
I hear you.